cpp/src/ray/runtime/task/invocation_executor.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+#include <ray/core.h>
+namespace ray {
+namespace api {
+
+class InvocationExecutor {
```

Comment:
This class is only used for local mode right? Can we just put it in `LocalModeTaskSubmitter`?

Reply:
It's more like `TaskExecutor` in Java. Why the logic isn't in `TaskExecutor`?

Reply:
Not only used to local mode, but also can be used to native mode. I can move to TaskExecutor.